ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/KeyValueParticipantStateWriter.scala

```diff
@@ -64,7 +64,7 @@ class KeyValueParticipantStateWriter(writer: LedgerWriter, metrics: Metrics) ext
     val submission =
       keyValueSubmission
         .configurationToSubmission(maxRecordTime, submissionId, writer.participantId, config)
-    commit(submissionId, submission)(NoOpTelemetryContext)
+    commit(submissionId, submission)
```

Comment:
Not sure why it's been working so far:
![image](https://user-images.githubusercontent.com/80257650/118301969-f1120d00-b4e3-11eb-8d4d-028628cc1d3c.png)

Reply:
And is it still working after this change? 😄

Reply:
I will have to test it e2e from the oem integration kit 😄

Reply:
I think it works, as we do not create any new spans and attributes since this place.